### PR TITLE
fix: wasm dev shell on macos

### PR DESCRIPTION
**Pull request by dpc**

### CODE REVIEW

**Comment by dpc** [comment]
@justinmoon please verify

File: flake.nix [hunk 1]
```diff
@@ -350,10 +353,10 @@
               });
 
               # Like `cross` but only with wasm
-              crossWasm = flakeboxLib.mkDevShell (craneMultiBuild.commonEnvsShell // craneMultiBuild.commonEnvsShellRocksdbLink // {
+              crossWasm = flakeboxLib.mkDevShell (commonShellArgs // {
                 toolchain = toolchainWasm;
 
-                packages = [
```
Comment (dpc) [inline flake.nix:356]: There's no such thing as `packages` in `mkShell`. :shrug:

File: nix/flakebox.nix [hunk 2]
```diff
@@ -137,9 +137,7 @@ let
       libiconv
       darwin.apple_sdk.frameworks.Security
       darwin.apple_sdk.frameworks.SystemConfiguration
-    ] ++ builtins.attrValues {
-      inherit (pkgs) openssl;
```
Comment (dpc) [inline nix/flakebox.nix:141]: duplicate

File: flake.nix [hunk 3]
```diff
@@ -10,7 +10,7 @@
     };
     flake-utils.url = "github:numtide/flake-utils";
     flakebox = {
-      url = "github:dpc/flakebox?rev=4c659e425a347e0e90fc661c48e7fb20014cc120";
+      url = "github:dpc/flakebox?rev=d7f57f94f2dca67dafd02b31b030b62f6fefecbc";
```
Comment (dpc) [inline flake.nix:13]: https://github.com/rustshop/flakebox/pull/128

**Comment by justinmoon** [comment]
I was able to run wasm tests on mac and they passed with `nix develop .#crossWasm --command ./scripts/tests/wasm-test.sh`

**Review by justinmoon — APPROVED** [review]

**Review by Maan2003 — APPROVED** [review]

**Pull request merged**